### PR TITLE
Release notes for Operators Updates

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: blog-service/2023-06-20-metrics.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Enhancements to rate & delta Operators (Metrics)
```
Comment (jpipkin1) [inline blog-service/2023-06-20-metrics.md:2]:
```suggestion
title: Enhancements to Rate and Delta Operators (Metrics)
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JainM6. The only change I have is to use uppercase for the title.

File: blog-service/2023-06-20-metrics.md (hunk 1 above)
Comment (JainM6) [reply blog-service/2023-06-20-metrics.md:2]: Thanks @jpipkin1 for the quick review. I have maintained consistency here as we have written the operators' titles everywhere in small caps. Sharing the reference link here. https://help.sumologic.com/docs/metrics/metrics-operators/delta/

Comment (JainM6) [reply blog-service/2023-06-20-metrics.md:2]: Let me know what do you suggest now.

Comment (jpipkin1) [reply blog-service/2023-06-20-metrics.md:2]: That sounds fine, @JainM6.

File: blog-service/2023-06-20-metrics.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - metrics
+  - rate & delta
```
Comment (kimsauce) [inline blog-service/2023-06-20-metrics.md:6]:
```suggestion
  - rate
  - delta
```

**Review by kimsauce — DISMISSED** [review]
minor fix

**Pull request merged**